agent/internal/agent.go

```diff
@@ -114,15 +120,19 @@ func (a *Agent) start() {
 
 			metrics := a.metricsMng.getObservations()
 			appBeatReq := api.AppBeatRequest{
-				Metrics: metrics,
+				Metrics:        metrics,
```

Comment:
To report in heartbeat as `command_results` and `metrics`

Reply:
Same as your previous comment I guess?